source/_patterns/templates/typography/vertical-rhythm.hbs

```diff
@@ -0,0 +1,32 @@
+<section class="container-fluid pf-verticalRhythm">
```

Comment:
A container or any layout classes should always be on their own, let's create a new div wrapper around the text you'd like to get vertical rythm.